examples/protocol_specification_ex/sample.yaml

```diff
@@ -25,11 +31,3 @@ speech_acts:
   decline: {}
   match_accept: {}
 ...
----
-ct:DataModel: |
```

Comment:
In my opinion you should at least add one custom type to the sample and test sample so we cover all cases.